pkg/nfd-master/nfd-master.go

```diff
@@ -54,6 +54,7 @@ import (
 
 	"sigs.k8s.io/node-feature-discovery/pkg/apihelper"
 	nfdv1alpha1 "sigs.k8s.io/node-feature-discovery/pkg/apis/nfd/v1alpha1"
+	nodefeaturerule "sigs.k8s.io/node-feature-discovery/pkg/apis/nfd/v1alpha1/nodefeaturerule"
```

Comment:
I think the alias is not necessary since you are using the same pkg name

Reply:
True, removed